tests/integration/test_manylinux.py

```diff
@@ -302,7 +305,7 @@ def test_build_wheel_with_image_dependencies(with_dependency, any_manylinux_cont
     assert 'manylinux' not in orig_wheel
 
     repair_command = \
-        'LD_LIBRARY_PATH=/auditwheel_src/tests/integration/testdependencies '\
+        'LD_LIBRARY_PATH=/auditwheel_src/tests/integration/testdependencies:$LD_LIBRARY_PATH '\
```

Comment:
This change seems unrelated. It's probably harmless, but I wonder why you are changing it.

Reply:
When running python 3.7 tests, `libcrypto.so.2` needs to be found. Overwriting `LD_LIBRARY_PATH` not preserving the existing prevents `libcrypto.so.2` to be found.